www/authoring/index.php

```diff
@@ -173,6 +173,22 @@
                 </div>
             </div>
         </div>
+        <div class="row">
+            <div class="col-md-6">
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <h2 class="panel-title">Workflow States and Comments</h2>
+                    </div>
+                    <div class="panel-body">
+                        <p>Move your item through a series of workflow states, from a first draft to the
+                            finished product.</p>
+                        <p class="text-right">
+                            <a class="demo_link"  href="./workflow.php">Demo</a>
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
```

Comment:
@klauste I'd suggest a minor change: moving this down to the "Creating Content, Questions, and Features" section just below for clarity. Right now it's in the "Browsing Items and Activities" section which seems better suited to item list/activity list related functionality.

Reply:
@hbalseiro I've moved the workflow demo as you suggested. Does it matter that the demo opens the item list page (that's on purpose as searching by workflow is possible)?

Reply:
@klauste No, that should work just fine. To be honest, we wouldn't object if Team Authoring felt the other section was the best place for it, it was sincerely just a suggestion. We're mainly concerned with keeping the code simple and readable so clients can understand it at a glance and yours was perfect in that sense.